src/Jobs/DeleteModelFromMautic.php

```diff
@@ -3,19 +3,18 @@
 namespace Swis\Laravel\Mautic\Jobs;
 
 use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Foundation\Bus\Dispatchable;
 use Illuminate\Queue\InteractsWithQueue;
-use Illuminate\Queue\SerializesModels;
 use Swis\Laravel\Mautic\Facades\Mautic;
 use Swis\Laravel\Mautic\SynchronizesWithMautic;
 
-class DeleteModelFromMautic
+class DeleteModelFromMautic implements ShouldQueue
 {
     use Dispatchable;
     use InteractsWithQueue;
     use Queueable;
-    use SerializesModels;
```

Comment:
This trait is very important if your job has actual models as properties! Which this job has...

Reply:
You now need to give the job the mauticID, type and connection in order to let it do it's thing.